docs/tutorials/dashboard.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Jellyfish Dashboard
+---
+
+import { ImageWrapper } from "./ImageWrapper";
+import StartingJellyfishBackend from "./common/_starting-jellyfish-backend.mdx";
+import StartingJellyfishDashboard from "./common/_starting-dashboard.mdx";
+
+# The Whats and Hows of Jellyfish Dashboard
+
+## What is Jellyfish Dashboard?
+
+Jellyfish Dashboard is a web application that allows you to manage your
+Jellyfish backend. It is a React application that uses the Jellyfish React Client API
+to communicate with the backend.
+
+It was created to help Jellyfish developers in testing but was later extended and
+adapted to as a tool for tutorials and demos. Here we present a short introduction to
+the dashboard and its features.
+
+## How to start Jellyfish Dashboard?
+
+<StartingJellyfishDashboard/>
+
+## How to start Jellyfish Backend?
+
+<StartingJellyfishBackend/>
```

Comment:
I just noticed, that in the docker command we are locked on a specific jellyfish version. I think instead we could put a placeholder here, like so:

"""
```bash
docker run -p 50000-50050:50000-50050/udp \
           .....
           ghcr.io/jellyfish-dev/<jellyfish-image-id>
```

For latest jellyfish version use `edge` image.
You can find all available packages here: https://github.com/jellyfish-dev/jellyfish/pkgs/container/jellyfish .
"""

Reply:
Right now, we need a specific version because it's too risky to use the `edge` version. In the future, we may change it to a placeholder or the edge version, but for now, we need a working pair of Jellyfish and Dashboard